vortexasdk/api/cargo_movement.py

```diff
@@ -57,9 +148,27 @@ class CargoMovement(FromDictMixin):
     """
 
     cargo_movement_id: ID
-    quantity: int
-    status: str
-    vessels: List[VesselEntity]
-    product: List[ProductEntity]
-    events: List[CargoEvent]
-    parent_ids: List[ParentID]
+    quantity: Optional[int] = None
+    status: Optional[str] = None
+    vessels: Optional[List[VesselEntity]] = None
+    product: Optional[List[CargoMovementProductEntry]] = None
+    parent_ids: Optional[List[ParentID]] = None
+    events: Optional[
+        List[
+            Annotated[
```

Comment:
What is the `Annotated` type?

Reply:
The Annotated type adds context specific metadata to a type. In this case, Pydantic is reading the metadata to know which field in the Model to use a discriminator. See docs here: https://pydantic-docs.helpmanual.io/usage/types/#discriminated-unions-aka-tagged-unions